entries/x/xda-developers.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Site Name": {
+    "domain": "xda-developers.com",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "categories": [
+      "social"
```

Comment:
What type of social service does the website provide?
Please note that we do not list forums so if that's the only service that requires authentication, then the site can't be listed.

Reply:
Oh I guess I missed that requirement. Yeah it's a forum, so we can close this :(